src/client/components/movies/Movies.js

```diff
@@ -27,19 +32,24 @@ class Movies extends React.Component {
       )
     }
     if (this.props.movies.length) {
-      const renderMovie = (movie) => { return <Movie key={movie.id} movie={movie} /> }
+      const renderMovie = (movie) => { 
+        const MovieContainer = withIdConnection(Movie);
+        return <MovieContainer key={movie.id} movieDescription={movie} /> 
+      }
       let moviesList = this.props.movies.map(renderMovie);
       return (
         <InfiniteScroll
           dataLength={this.props.movies.length}
           next={() => {
```

Comment:
it would cause re-render on each moment since always create new object (function is object). Please memoize this function